src/test/java/io/jenkins/plugins/analysis/core/model/IconLabelProviderTest.java

```diff
@@ -0,0 +1,51 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import org.junit.jupiter.api.Test;
+
+import static io.jenkins.plugins.analysis.core.assertions.Assertions.*;
+
+/**
+ * Tests the class {@link IconLabelProvider}.
+ *
+ * @author Kezhi Xiong
+ */
+class IconLabelProviderTest {
+    private static final String iconId = "icon-id";
+    private static final String toolName = "tool-name";
+    private static final String iconName = "icon-name";
+
+    /**
+     * This field has to be the same as in IconLabelProvider. Create this
+     * to avoid using reflection or adding getters in the origin class.
+     */
+    private static final String ICONS_URL = "/plugin/warnings-ng/icons/";
+
+    private static final String smallIconUrlSuffix = "-24x24.png";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The constant name 'smallIconUrlSuffix' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5011839)